pkg/kafka/topic/validators.go

```diff
@@ -81,6 +86,10 @@ func (v *Validator) ValidatePartitionsN(val interface{}) error {
 		return errors.New(v.Localizer.MustLocalize("kafka.topic.common.validation.partitions.error.invalid.minValue", localize.NewEntry("Partitions", partitions), localize.NewEntry("Min", minPartitions)))
 	}
 
+	if partitions < v.CurPartitions {
```

Comment:
Equality also might not make sense? 
Cna we do <= ?

Reply:
Making a patch with the same number of partitions doesn't throw any errors from server, hence I resorted to not using equality here, wdyt?

Reply:
ok. no issue here